java/ql/test/experimental/query-tests/security/CWE-074/options

```diff
@@ -0,0 +1 @@
+//semmle-extractor-options: --javac-args -cp ${testdir}/../../../stubs/springframework-5.2.3:${testdir}/../../../stubs/Saxon-HE-9.9.1-7
```

Comment:
```suggestion
//semmle-extractor-options: --javac-args -cp ${testdir}/../../../../stubs/springframework-5.2.3:${testdir}/../../../stubs/Saxon-HE-9.9.1-7
```

Reply:
Might as well reuse the existing spring stubs.

Reply:
I've changed it to reuse the existing stubs.